tka/state.go

```diff
@@ -36,6 +36,13 @@ type State struct {
 
 	// Keys are the public keys currently trusted by the TKA.
 	Keys []Key `cbor:"3,keyasint"`
+
+	// StateID's are nonce's, generated on enablement and fixed for
+	// the lifetime of the Tailnet Key Authority. We generate 16-bytes
+	// worth of keyspace here just in case we come up with a cool future
+	// use for this.
+	StateID1 uint64 `cbor:"4,keyasint,omitempty"`
```

Comment:
btw these are omitempty so I don't have to update all the tests that depend on specific hash values.

Reply:
Any particular reason do this as `uint64` instead of `[16]byte` or `[]byte`?

Reply:
No strong reason: I just thought it would be easier to use as a `uint64` and it also encodes a bit nicer as CBOR.